gef.py

```diff
@@ -5688,7 +5689,8 @@ class SearchPatternCommand(GenericCommand):
     _aliases_ = ["grep", "xref"]
     _example_ = (f"\n{_cmdline_} AAAAAAAA"
                  f"\n{_cmdline_} 0x555555554000 little stack"
-                 f"\n{_cmdline_} AAAA 0x600000-0x601000")
+                 f"\n{_cmdline_} AAAA 0x600000-0x601000"
+                 f"\n{_cmdline_} --regex 0x401000 0x401500 ([\\\\x20-\\\\x7E]{{2,}})(?=\\\\x00)   <-- It matchs null-end-printable(from x20-x7e) C strings (min size 2 bytes)")  
```

Comment:
FWIW since last release `_example_` now also accepts `List[str]` type so you can replace with a clearer version:

```python
    _example_ = [f"{_cmdline_} AAAAAAAA",
                 f"{_cmdline_} 0x555555554000 little stack",
                 f"{_cmdline_} AAAA 0x600000-0x601000",
                 f"{_cmdline_} --regex 0x401000 0x401500 ([\\\\x20-\\\\x7E]{{2,}})(?=\\\\x00)   <-- It matchs null-end-printable(from x20-x7e) C strings (min size 2 bytes)"]
```

Reply:
done!